### PR TITLE
Error page

### PR DESCRIPTION
**Pull request by matejoslav**

### CODE REVIEW

File: web-app/src/components/errors/ErrorBoundary.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import ErrorPage from "../../pages/errorPage";
+
+export default class ErrorBoundary extends React.Component {
```
Comment (MadelinTom) [inline web-app/src/components/errors/ErrorBoundary.js:4]: Does this need to be a class component? Would be nice if it was a functional component to keep things consistent

**Review by MadelinTom — CHANGES_REQUESTED** [review]

File: web-app/src/components/errors/ErrorBoundary.js (hunk 1 above)
Comment (matejoslav) [reply web-app/src/components/errors/ErrorBoundary.js:4]: https://reactjs.org/docs/error-boundaries.html#introducing-error-boundaries

"Only class components can be error boundaries."

There is a package which makes it possible to have a func component as the handler but its not worth imo (npm i react-error-boundary)

Comment (MadelinTom) [reply web-app/src/components/errors/ErrorBoundary.js:4]: Didn't know that! Looks good then :)

**Review by MadelinTom — APPROVED** [review]

**Comment by arnoldst** [comment]
IBM Cloud [toolchain](https://cloud.ibm.com/devops/toolchains/0a72695f-1fa5-4df5-9d98-aaa218c70126?env_id=ibm:yp:eu-de "lrp-toolchain"): [Delivery Pipeline](https://cloud.ibm.com/devops/pipelines/66e6d48f-2aa6-41b2-a82d-e4fac4a78dc9 "lrp-toolchain") deployed [LRP-web-app](https://github.com/lebanon-relief-project/LRP-web-app/releases/tag/deploy-ibm-yp-eu-de-Lebanon_Relief_Network-dev-20210722-093233 "deploy-ibm-yp-eu-de-Lebanon_Relief_Network-dev-20210722-093233") to [dev](https://console.bluemix.net/apps/3f8ccb74-4417-451f-824b-a0e1d349eaa9?env_id=ibm:yp:eu-de "ibm:yp:eu-de:Lebanon_Relief_Network:dev"), including commit ff386362638fbdc301cfd7891bd194f9677de1eb

**Comment by arnoldst** [comment]
IBM Cloud [toolchain](https://cloud.ibm.com/devops/toolchains/0a72695f-1fa5-4df5-9d98-aaa218c70126?env_id=ibm:yp:eu-de "lrp-toolchain"): [Delivery Pipeline](https://cloud.ibm.com/devops/pipelines/66e6d48f-2aa6-41b2-a82d-e4fac4a78dc9 "lrp-toolchain") deployed [LRP-web-app](https://github.com/lebanon-relief-project/LRP-web-app/releases/tag/deploy-ibm-yp-eu-de-Lebanon_Relief_Network-test-20210724-155957 "deploy-ibm-yp-eu-de-Lebanon_Relief_Network-test-20210724-155957") to [test](https://console.bluemix.net/apps/c8ab7a12-26a8-4221-9d60-ce65d790011e?env_id=ibm:yp:eu-de "ibm:yp:eu-de:Lebanon_Relief_Network:test"), including commit ff386362638fbdc301cfd7891bd194f9677de1eb

**Comment by arnoldst** [comment]
IBM Cloud [toolchain](https://cloud.ibm.com/devops/toolchains/0a72695f-1fa5-4df5-9d98-aaa218c70126?env_id=ibm:yp:eu-de "lrp-toolchain"): [Delivery Pipeline](https://cloud.ibm.com/devops/pipelines/66e6d48f-2aa6-41b2-a82d-e4fac4a78dc9 "lrp-toolchain") deployed [LRP-web-app](https://github.com/lebanon-relief-project/LRP-web-app/releases/tag/deploy-ibm-yp-eu-de-Lebanon_Relief_Network-prod-20210724-163313 "deploy-ibm-yp-eu-de-Lebanon_Relief_Network-prod-20210724-163313") to [prod](https://console.bluemix.net/apps/e446a03a-f8e7-4e88-8061-eaf9eba0bb19?env_id=ibm:yp:eu-de "ibm:yp:eu-de:Lebanon_Relief_Network:prod"), including commit ff386362638fbdc301cfd7891bd194f9677de1eb

**Pull request merged**